manifests/m/Microsoft/PowerShell/7.3.4.0/Microsoft.PowerShell.locale.en-US.yaml

```
@@ -0,0 +1,37 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=NVS1.CRLF.7-3-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.3.4.0
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://github.com/PowerShell/PowerShell/
+PublisherSupportUrl: https://github.com/PowerShell/PowerShell/issues
+# PrivacyUrl:
+Author: Microsoft Corporation
+PackageName: PowerShell
+PackageUrl: https://microsoft.com/PowerShell
+License: MIT
+LicenseUrl: https://github.com/PowerShell/PowerShell/blob/master/LICENSE.txt
+Copyright: Copyright (c) Microsoft Corporation
+CopyrightUrl: https://github.com/PowerShell/PowerShell/blob/master/LICENSE.txt
+ShortDescription: PowerShell is a cross-platform task automation solution made up of a command-line shell, a scripting language, and a configuration management framework. PowerShell runs on Windows, Linux, and macOS.
+Description: |-
+  PowerShell is a cross-platform (Windows, Linux, and macOS) automation and configuration tool/framework that works well with your existing tools and is optimized for dealing with structured data (e.g. JSON, CSV, XML, etc.), REST APIs, and object models.
+  It includes a command-line shell, an associated scripting language and a framework for processing cmdlets.
+Moniker: pwsh
+Tags:
+- command-line
+- cross-platform
+- open-source
+- powershell
+- pwsh
+- shell
+# Agreements:
+# ReleaseNotes:
+ReleaseNotesUrl: https://github.com/PowerShell/PowerShell/releases/tag/v7.3.4
+# PurchaseUrl:
+# InstallationNotes:
+# Documentations:
```

```suggestion
Documentations:
- DocumentLabel: Product Documentation
  DocumentUrl: https://learn.microsoft.com/powershell
- DocumentLabel: FAQ
  DocumentUrl: https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md
```
Metadata to make the manifest more complete if you wish